docker/docker-compose.example.yml

```diff
@@ -48,3 +48,41 @@ services:
       MAGPIE_POSTGRES_DB: magpie
       MAGPIE_POSTGRES_PORT: 5432
     restart: always
+
+  geoserver:
+    image: kartoza/geoserver:2.19.0
+    container_name: geoserver
+    ports:
+      - "8765:8080"
+    restart: on-failure
+    volumes:
+      - /user_workspaces:/user_workspaces
```

Comment:
See cowbird here : https://github.com/Ouranosinc/cowbird/blob/master/docker/docker-compose.override.example.yml#L20
A local directory was used to avoid requiring that directory on the dev machine. (It is even commited (https://github.com/Ouranosinc/cowbird/tree/master/docker/user_workspaces), but sadly I saw that some artefacts got commited there, a .gitignore should be added to avoid comitting user directory)

Reply:
I think it would be better to set host path on `/tmp/user_workspaces` to avoid dumping directories all over the place. User should provide a specific override (or edit file) if it wants to persist data from it.

Reply:
I had a talk with @dbyrns on this. I ended up replacing everything with environment variables, as discussed elsewhere, but in https://github.com/Ouranosinc/cowbird/blob/geoserver-setup/docker/.env.example I did set it to the local directory mentioned by him as we both found it was easier for manual testing.

Reply:
I agree with using variables to simplify setup without the need to edit any YML file. 
I still disagree with using (default) local relative paths though. 
It should be a tmp dir, which both locally for testing or in CI can be overridden with the variable.
This is to avoid mistake commits such as https://github.com/Ouranosinc/cowbird/tree/master/docker/user_workspaces